lasagne/layers/recurrent.py

```diff
@@ -0,0 +1,1239 @@
+"""
+Layers to construct recurrent networks. Recurrent layers can be used similarly
+to feed-forward layers except that the input shape is expected to be
+``(batch_size, sequence_length, num_inputs)``. The input is allowed to have
+more than three dimensions in which case dimensions trailing the third
+dimension are flattened.
```

Comment:
This sounds as if convolutional recurrent layers were outright impossible. They should be possible by providing custom input-to-hidden and hidden-to-hidden networks, shouldn't they?